src/Model.hs

```diff
@@ -40,6 +42,7 @@ instance FromJSON Command where
     parseJSON = withObject "Command" $ \o -> Command
         <$> o .: "running"
         <*> o .: "description"
+        <*> o .:? "archived" .!= False
```

Comment:
Yup. Since I've currently got commands in the DB (serialized JSON in Redis) without this field, I have to default it when I read them out.